client/client.go

```diff
@@ -271,6 +271,9 @@ func (o *OpkClient) oidcAuth(
 	return pkt, nil
 }
 
+// Refresh is used for refreshing the ID token and access token for the OIDC provider using the refresh token.
```

Comment:
Nit: OIDC Provider referred to in the spec as the OpenID Provider.

```suggestion
// Refresh is used for refreshing the ID token and access token for the OpenID provider using the refresh token.
```